frontend/src/pages/account/NoTeamsUser.vue

```diff
@@ -1,20 +1,18 @@
 <template>
-    <main>
```

Comment:
Remove the `main` found in one of the child elements, as the parent (`Home`) now handles this